lib/services/project-data-service.ts

```diff
@@ -63,7 +63,10 @@ export class ProjectDataService implements IProjectDataService {
 					"description": "NativeScript Application",
 					"license": "SEE LICENSE IN <your-license-filename>",
 					"readme": "NativeScript Application",
-					"repository": "<fill-your-repository-here>"
+					"repository": "<fill-your-repository-here>",
+					"devDependencies": {
```

Comment:
I can move it to the templates, but are we sure there aren't any templates outside the NativeScript organization? Or should I add it to the hello-world template and it will be copied like the App_Resources are?